Rakefile

```diff
@@ -142,6 +142,29 @@ namespace :license do
   end
 end
 
+namespace :changelog do
+  desc 'Updates the changelog with commit messages'
+  task :update, [:new_version] do |t, args|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `t`. If it's necessary, use `_` or `_t` as an argument name to indicate that it won't be used.